src/main/java/org/skriptlang/skript/registry/DefaultSyntaxInfos.java

```diff
@@ -0,0 +1,89 @@
+/**
+ *   This file is part of Skript.
+ *
+ *  Skript is free software: you can redistribute it and/or modify
+ *  it under the terms of the GNU General Public License as published by
+ *  the Free Software Foundation, either version 3 of the License, or
+ *  (at your option) any later version.
+ *
+ *  Skript is distributed in the hope that it will be useful,
+ *  but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ *  GNU General Public License for more details.
+ *
+ *  You should have received a copy of the GNU General Public License
+ *  along with Skript.  If not, see <http://www.gnu.org/licenses/>.
+ *
+ * Copyright Peter Güttinger, SkriptLang team and contributors
+ */
+package org.skriptlang.skript.registry;
+
+import ch.njol.skript.lang.ExpressionType;
+import ch.njol.skript.lang.SkriptEvent;
+import org.jetbrains.annotations.ApiStatus;
+import org.jetbrains.annotations.Contract;
+import org.jetbrains.annotations.Nullable;
+import org.skriptlang.skript.lang.entry.EntryValidator;
+
+import java.util.List;
+
+@ApiStatus.Internal
+interface DefaultSyntaxInfos {
+	
+	@ApiStatus.NonExtendable
+	interface Event<E extends SkriptEvent> extends Structure<E> {
```

Comment:
Events are a Bukkit concept and thus should be separated from this. You could move this to a class like `org.skriptlang.skriptbukkit.registration.BukkitSyntaxInfos`